rpc/tests/p2p.rs

```diff
@@ -0,0 +1,296 @@
+use crate::utils::client::{new_test_client, AuthLevel};
+use celestia_rpc::p2p;
+use celestia_rpc::prelude::*;
+use tokio::time::{sleep, Duration};
+
+pub mod utils;
+
+async fn rpc_call_delay() {
+    // delay for RPC calls like connect/close to let node finish the operation before we query it
+    // again. Below 150 ms I start getting intermittent failures.
+    sleep(Duration::from_millis(150)).await;
+}
+
+#[tokio::test]
+async fn info_test() {
+    let client = new_test_client(AuthLevel::Admin).await.unwrap();
+    client.p2p_info().await.expect("Failed to get node info");
+}
+
+#[tokio::test]
+async fn add_remove_peer_test() {
+    //! add and then remove a peer, testing outputs from `p2p.Peers` and `p2p.Connectedness`
```

Comment:
use `//`, the `//!` is a doc-comment that describes the whole file, and they should be only on the beginning of the file. you have it in other places too

Reply:
I wanted to do a doc-comment that refers to the outer block (i.e. the test function), but I think I agree it makes sense to switch them to regular comments.

Reply:
in that case i would prefer the `///` doc-comment above the function